src/pages/blogs/[id].tsx

```diff
@@ -22,14 +23,14 @@ export async function getStaticPaths() {
 
 export default function Post({ postData }) {
     return (
-        <div>
-            <h1>{postData.title}</h1>
+        <Box>
```

Comment:
Add `padding: '0 7.5% 100px'`  same as the Index page

Reply:
completed styling of blog post page and blog list page.



![Screenshot (109)](https://github.com/AMC-website/Frontend/assets/138979034/68534899-2195-4d3f-96f3-bd676f96ec92)
![Screenshot (110)](https://github.com/AMC-website/Frontend/assets/138979034/a1e2571d-70ee-4c51-bcd4-197f39aad2bf)